Assets/Scripts/Models/Parameter.cs

```diff
@@ -222,9 +224,16 @@ public bool HasContents()
 
     public void WriteXmlParamGroup(XmlWriter writer)
     {
-        writer.WriteStartElement("Param");
-        writer.WriteAttributeString("name", name);
-        if (string.IsNullOrEmpty(value) == false)
+        if (string.IsNullOrEmpty(name))
+        {
+            writer.WriteStartElement("Params");
+        }
+        else
+        {
+            writer.WriteStartElement("Param");
+            writer.WriteAttributeString("name", name);
+        }
+        if (!(value == null) )
```

Comment:
dangit, one escaped me... And I'm sure I had a reason to write the null check like that... it just may have been insanity however.